youtube_dl/__init__.py

```diff
@@ -382,8 +382,6 @@ def parse_retries(retries):
         'external_downloader_args': external_downloader_args,
         'postprocessor_args': postprocessor_args,
         'cn_verification_proxy': opts.cn_verification_proxy,
-        'geo_verification_proxy': opts.geo_verification_proxy,
```

Comment:
Sorry, I did not mean to delete that line. Later today when I can get to my dev system, I'll correct that.